back-end/api/quickstart/models.py

```diff
@@ -48,11 +48,10 @@ class Visibility(models.TextChoices):
 # An Author can communicate with other Authors by commenting on posts. 
 class Comment(models.Model):
   id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+  post = models.ForeignKey(Post, on_delete=models.CASCADE)
   type = 'comment'
-  postId = models.TextField()
-  author = models.TextField()
+  author = models.JSONField()
   comment = models.TextField()
-  contentType = models.TextField()
```

Comment:
Is it alright to remove this? Are comments just gonna be plain text?

Reply:
Markdown support for comments doesn't seem to be in the requirements, so it seems that we don't need this.

Reply:
Awesome sounds good!